src/layouts/index.js

```diff
@@ -112,14 +136,31 @@ const HeaderMobile = ({ language, hasLanguage, setLanguage }) => {
           <Link to="/what-is-ebpf">What is eBPF?</Link>
           <Link to="/blog">Blog</Link>
           <Link to="/projects">Projects</Link>
+          <span className="languageSelect">
+            <button className="button" onClick={() => setIsConferencesMenuShown(!isConferencesMenuShown)} type="button">Conferences <span className="triangle">▾</span></button>
+            <span className={`list${isConferencesMenuShown ? ' is-shown' : ''}`}>
+              <strong>eBPF Summit</strong>
+              <Link className="link" to="/summit-2021">2021</Link>
+              <Link className="link" to="/summit-2020">2020</Link>
+              <strong>Linux Plumbers</strong>
+              <a target="_blank" className="link" href="https://lore.kernel.org/bpf/6d225920-9ecc-ef24-2bf8-848ca86c7fb0@iogearbox.net/">2021</a>
+              <a target="_blank" className="link" href="https://linuxplumbersconf.org/event/7/sessions/91/#all">2020</a>
+              <a target="_blank" className="link" href="https://linuxplumbersconf.org/event/4/sessions/62/#20190911">2019</a>
+              <a target="_blank" className="link" href="http://vger.kernel.org/lpc-bpf2018.html">2018</a>
+              <a target="_blank" className="link" href="https://blog.linuxplumbersconf.org/2017/ocw/events/LPC2017/schedule.html#day_2017_09_15">2017</a>
+              <strong>bpfconf</strong>
+              <a target="_blank" className="link" href="https://lore.kernel.org/bpf/60997079.1c69fb81.77f3f.a045@mx.google.com/">2021</a>
+              <a target="_blank" className="link" href="http://vger.kernel.org/bpfconf2019.html">2019</a>
+            </span>
+          </span>
```

Comment:
These headers (for the mobile version) are not updated in your second commit (only the desktop version).

It looks like the menu is the same on both version, it might be worth factoring that into a function that would generate the HTML at both places? I don't really mind, but either way let's just make sure that both versions of the menu are consistent.